README.md

```diff
@@ -1,66 +1,48 @@
-
-## Packages
-
-The Primer repo is managed as a monorepo that is composed of many npm packages.
-
-### Core Packages
-
-| Package | Version |
-|---|---|
-| **[primer](/modules/primer)** <br />Includes all 23 packages | [![npm](https://img.shields.io/npm/v/primer.svg)](https://www.npmjs.com/package/primer) |
-| [primer-core](/modules/primer-core) | [![npm](https://img.shields.io/npm/v/primer-core.svg)](https://www.npmjs.com/package/primer-core) |
-| [primer-product](/modules/primer-product) |  [![npm](https://img.shields.io/npm/v/primer-product.svg)](https://www.npmjs.com/package/primer-product) |
-| [primer-marketing](/modules/primer-marketing) | [![npm](https://img.shields.io/npm/v/primer-marketing.svg)](https://www.npmjs.com/package/primer-marketing) |
+Primer CSS is the Sass implementation of [Primer], the design system that powers GitHub.
 
 ## Install
-
-This repository is distributed with [npm][npm]. After [installing npm][install-npm], you can install `primer` with this command.
+This repository is distributed with [npm]. After [installing npm][install-npm], you can install `@primer/css` with this command:
 
 ```sh
-$ npm install --save primer
+$ npm install --save @primer/css
 ```
 
 ## Usage
-
-The source files included are written in [Sass][sass] (SCSS). After [installing](#install) with npm, you can add your project's `node_modules` directory to your Sass [include paths](https://github.com/sass/node-sass#includepaths) (AKA [load paths](http://technology.customink.com/blog/2014/10/09/understanding-and-using-sass-load-paths/) in Ruby), then import it like this:
+The included source files are written in [Sass] using SCSS syntax. After [installing](#install) with npm, you can add your project's `node_modules` directory to your Sass [include paths](https://github.com/sass/node-sass#includepaths) (AKA [load paths](http://technology.customink.com/blog/2014/10/09/understanding-and-using-sass-load-paths/) in Ruby), then import it like this:
 
 ```scss
-@import "primer/index.scss";
+@import "@primer/css/index.scss";
 ```
 
-You can import individual Primer modules by installing them with npm, for instance:
-
-```sh
-$ npm install --save primer-navigation
-```
-
-Then, you would import the module with:
+You can import individual Primer modules directly from the `@primer/css` package:
 
 ```scss
-@import "primer-navigation/index.scss";
+@import "@primer/css/core/index.scss";
+@import "@primer/css/product/index.scss";
+@import "@primer/css/marketing/index.scss";
 ```
 
 ## Development
 See [DEVELOP.md](./DEVELOP.md) for development docs.
 
-## Releasing (Staff only)
+## Releasing (for GitHub staff)
 You can find docs about our release process in [RELEASING.md](./RELEASING.md).
 
 ## Documentation
-
-Primer CSS documentation is published to the [GitHub Style Guide](https://styleguide.github.com/primer/).
+The Primer CSS docs site in this repo is created with [Primer Components], [Next], and hosted on [Now].
```

Comment:
This section should link to the documentation site instead of describing how it's built IMO.